frontend/src/router.js

```diff
@@ -10,8 +10,10 @@ const NavigationAuth = () =>
   import(/* webpackChunkName: "navigationAuth" */ './views/auth/NavigationAuth.vue')
 const NavigationDefault = () =>
   import(/* webpackChunkName: "navigationDefault" */ './views/NavigationDefault.vue')
-const NavigationCamp = () =>
-  import(/* webpackChunkName: "navigationCamp" */ './views/camp/NavigationCamp.vue')
+const CampNavigation = () =>
+  import(
+    /* webpackChunkName: "navigationCamp" */ './views/camp/navigation/CampNavigation.vue'
```

Comment:
If you keep the renamed component, please change the webpack chunk name as well (actually, does that even do anything anymore with vite?)